upd_script/fetch.sh

```diff
@@ -90,6 +90,11 @@ update_gopigo() {
         feedback "##############################"
         curl -kL https://raw.githubusercontent.com/DexterInd/GoPiGo3/$selectedbranch/Install/update_gopigo3.sh | sudo -u pi bash -s -- --bypass-rfrtools  $selectedbranch
 
+        # set wifi antenna led for Raspbian for Robots
+        sudo mkdir /etc/systemd/system/antenna_wifi.service.d/
```

Comment:
It has to be `sudo mkdir -p ...`.

Reply:
I guess for safety but if /etc/systemd/system doesn't already exist, we have bigger problems